transifex/pull.py

```diff
@@ -93,4 +102,4 @@ def parse_arguments():
 
         with concurrent.futures.ProcessPoolExecutor() as executor:
```

Comment:
Like I noted in https://github.com/edx/ecommerce-scripts/pull/55, you can just get rid of this block.